cli/flox/src/config/features.rs

```diff
@@ -10,7 +10,16 @@ pub struct Features {
     #[serde(default)]
     pub search_strategy: SearchStrategy,
     #[serde(default)]
-    pub use_catalog: bool,
+    pub use_catalog: UseCatalog,
+}
+
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, derive_more::Deref)]
+pub struct UseCatalog(bool);
+
+impl Default for UseCatalog {
+    fn default() -> Self {
+        UseCatalog(true)
+    }
```

Comment:
nonblocking: I think I slightly prefer `#[serde(default="return_true")]`,
```
fn return_true() -> bool {
    true
}
```

Reply:
Doesn't work unfortunately if we use unwrap_or_default where default is the default impl of bool I.e false